Libplanet/Net/PeerState.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+
+namespace Libplanet.Net
+{
+    public class PeerState
```

Comment:
It would be better to be `internal`, or have docs (i.e., XML comments).

Reply:
Now `Swarm<T>` has `PeerStates` as its parameter, it will be great to have docs.